torchvision/transforms/v2/_geometry.py

```diff
@@ -593,6 +593,11 @@ class RandomRotation(Transform):
             Note that the expand flag assumes rotation around the center and no translation.
         center (sequence, optional): Optional center of rotation, (x, y). Origin is the upper left corner.
             Default is the center of the image.
+
+            ..note::
+
+              In theory, ``center`` has no effect together with ``expand=True``. In practice however, this can lead to
+              off-by-one differences of the resulting image size compared to ``center=None``.
```

Comment:
"off-by-one" can be a bit misleading because the numerical value will usually be much higher than 1 right? It's the pixels that are off, but their values can be way larger (e.g. on edges). Not sure how to accurately convey that though.

Also, should just add something like "See note below about ``center``" in the ``expand`` section?

Reply:
If you remove the warning, add a sentence to center docstring that if provided center together with expand=True, expected behaviour of the rotation op should be such that the pivot point is in the image center.
IMO, words like "in theory ... but in practice" do not look appropriate here, I think it is numerical precision problem